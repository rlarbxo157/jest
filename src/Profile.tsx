import React from 'react'

interface UserInfoProps{
    username:string;
    name:string;
}

const Profile :React.FunctionComponent<UserInfoProps> = (props) => {
    return (
        <>
            <div>
                <b>
                    {props.username}
                </b>
                <span>
                    ({props.name})
                </span>
            </div>
        </>
    );
};

export default Profile;