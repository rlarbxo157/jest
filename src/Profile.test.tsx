import React from 'react';
// import {render} from 'react-testing-library';
import {render} from '@testing-library/react';
import Profile from './Profile';

describe('<Profile />',()=> {
    it('matches snapshot',()=> {
        const utils = render(<Profile username="김규태" name="김규태123" />);
        expect(utils.container).toMatchSnapshot();
    })
})