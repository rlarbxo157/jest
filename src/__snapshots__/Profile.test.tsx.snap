// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile /> matches snapshot 1`] = `
<div>
  <div>
    <b>
      김규태
    </b>
    <span>
      (
      김규태123
      )
    </span>
  </div>
</div>
`;
